src/redux/books/books.js

```diff
@@ -0,0 +1,69 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = {
+  booksList: [
+    {
+      aboutBook: {
+        id: '1',
+        title: 'The Hunger Games',
+        author: 'Suzanne Collins',
+        category: 'action',
+      },
+      bookProgress: {
+        chapter: 17,
+        chapterName: null,
+        completed: 65,
+      },
+    },
+    {
+      aboutBook: {
+        id: '2',
+        title: 'Dune',
+        author: 'Frank Herbert',
+        category: 'science fiction',
+      },
+      bookProgress: {
+        chapter: 3,
+        chapterName: 'A Lesson Learned',
+        completed: 8,
+      },
+    },
+    {
+      aboutBook: {
+        id: '3',
+        title: 'Capital in the 21st Century',
+        author: 'Suzanne Collins',
+        category: 'economy',
+      },
+      bookProgress: {
+        chapter: null,
+        chapterName: 'Introduction',
+        completed: 0,
+      },
+    },
+  ],
+};
```

Comment:
- [ ] Your initial state should be an empty `Array` as the project requirements said, please delete these book objects 🚀 
    - A list of default books is no longer required for this milestone 😄
    ![imagen](https://user-images.githubusercontent.com/60789317/136590914-1d0b6bd8-5165-4bf5-a4c2-57eda99338ad.png)